apps/backend/src/lib/controller/postgres/prisma/message/message.ts

```diff
@@ -0,0 +1,46 @@
+import { PrismaPostgresClient } from "@/lib/controller/postgres/prisma/client";
+import { MessageData, MessageDataSchema, CreateMessageData } from "@types";
+
+PrismaPostgresClient.prototype.GetMessagesForUser = async function (
+  userSignaturePublicKey: string,
+  lastMessageFetchedAt: Date | undefined
+): Promise<MessageData[]> {
```

Comment:
What happens if there are no messages? Does `findMany` default to empty array or is it undefined / null?

Reply:
empty array